tests/datatypes/test_bytes.py

```diff
@@ -231,6 +231,16 @@ def test_contains(self):
             print(['b', 'e'] in b'pybee') #Test TypeError invalid byte array
         """, exits_early=True)
 
+    def test_center(self):
+        self.assertCodeExecution("""
+            print(b'pybee'.center(5))
+            print(b'pybee'.center(True, b'a'))
+            print(b'pybee'.center(13, b'a'))
```

Comment:
There's a lot more cases to test here:
* size < length of string
* error handling for non-bytestring fill arguments
* even and odd sizes for both input strings and final string size
* negative size values
* empty bytestrings